src/searchengine/nova/engines/torlock.py

```diff
@@ -0,0 +1,97 @@
+#VERSION: 2.0
+#AUTHORS: Douman (custparasite@gmx.se)
+#CONTRIBUTORS: Diego de las Heras (ngosang@hotmail.es)
+
+from novaprinter import prettyPrinter
+from helpers import retrieve_url, download_file
+from re import compile as re_compile
+from HTMLParser import HTMLParser
+
+class torlock(object):
+    url = "https://www.torlock.com"
+    name = "TorLock"
+    supported_categories = {'all'      : 'all',
+                             'anime'    : 'anime',
+                             'software' : 'software',
```

Comment:
That's strange.
Even PEP8 isn't supposed to complain about it.
And my original code used it like that.

Though this way might be more nice:
```
        self.supported_categories = {
                'all'      : 'all',
                'anime'    : 'anime',
                'software' : 'software',
                'games'    : 'game',
                'movies'   : 'movie',
                'music'    : 'music',
                'tv'       : 'television',
                'books'    : 'ebooks'
                }
```